index.js

```diff
@@ -31,7 +66,30 @@ function betweenExtremes() {}
  * Example: "A new month"
  * .- / -. . .-- / -- --- -. - ....
  */
-function morseCodeTranslator() {}
+function morseCodeTranslator(message, dictionary) {
+  const words = message.toUpperCase().split(' ');
+  const translatedMessage = [];
+
+  for (let i = 0; i < words.length; i++) {
+    const word = words[i];
+    const translatedWord = [];
+
+    for (let j = 0; j < word.length; j++) {
+      const char = word[j];
+
+      if (dictionary[char]) {
```

Comment:
Great work here - this is another pattern that is very efficient when it comes to quick data storage / retrieval.